From 63ff1c547a9c5b69b01cadd5d89c8299b9bc8c40 Mon Sep 17 00:00:00 2001
From: kongfl888 K <kongfl888@outlook.com>
Date: Mon, 20 Jul 2020 01:50:55 +0800
Subject: [PATCH 38/39] rk3328: add into supported_devices

Signed-off-by: kongfl888 K <kongfl888@outlook.com>
---
 target/linux/rockchip-rk3328/image/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/target/linux/rockchip-rk3328/image/Makefile b/target/linux/rockchip-rk3328/image/Makefile
index 203f0ea..d7c6bca 100644
--- a/target/linux/rockchip-rk3328/image/Makefile
+++ b/target/linux/rockchip-rk3328/image/Makefile
@@ -18,4 +18,6 @@ define Image/Build
 	cp $(KDIR)/vmlinux $(BIN_DIR)/$(IMG_PREFIX)-vmlinux
 endef
 
+SUPPORTED_DEVICES :=friendlyelec,nanopi-r2
+
 $(eval $(call BuildImage))
-- 
2.14.1.windows.1

